### DISCUSSION TITLE
When row created, create a row in another table

### DISCUSSION THREAD

**Discussion by IzakJackson** [supabase/supabase#11038] [Questions] [answered]
Hey,

I have a table and when I create a new entry I want to create an entry in a different table, that also stores the ID of the original entry.

I'd preferably want this to happen in the backend (so without submitting multiple API requests).

How would I go about doing this please?

As a follow on, if I updated a row in a table, how could I update a row in another table that has a column with the original rows ID?

**Comment by GaryAustin1** [comment, marked as answer]
This is standard Postgres stuff, so lots of info on web.
Googling this `postgres create row in another table trigger` shows quite a few hits.
This one  shows an example:
https://stackoverflow.com/questions/12343984/insert-trigger-to-update-another-table-using-postgresql

**Reply by IzakJackson** [reply]
Thanks @GaryAustin1 - I'll check out that post
